turbinia/jobs/__init__.py

```diff
@@ -14,12 +14,23 @@
 # limitations under the License.
 """Turbinia jobs."""
 
+import logging
 import uuid
 
+log = logging.getLogger('turbinia')
 
-def get_jobs():
+
+def get_jobs(jobs_blacklist=None, jobs_whitelist=None, jobs_list=None):
   """Gets a list of all job objects.
 
+  Only one of jobs_blacklist and jobs_whitelist can be specified at a time,
+  and all Jobs will be returned if both are specified.
+
+  Args:
+    jobs_blacklist (list): Jobs that will be excluded from running
```

Comment:
As above, please indicate the type of things in these lists

Reply:
Done.